tests/Constraint/EmptyConstraintTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of composer/semver.
+ *
+ * (c) Composer <https://github.com/composer>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Composer\Semver\Constraint;
+
+class EmptyConstraintTest extends \PHPUnit_Framework_TestCase
+{
+    protected $versionProvide;
+    protected $emptyConstraint;
+
+    protected function setUp()
+    {
+        $this->versionProvide = new Constraint('==', '1.1');
+        $this->emptyConstraint = new EmptyConstraint();
+    }
+
+    protected function tearDown()
+    {
+        unset($this->versionProvide);
+        unset($this->emptyConstraint);
```

Comment:
you should not unset the class properties. You should only reset their values (or do nothing as there is no circular dependency with the TestCase, so the refcounting will work fine)